From: Lukasz Skalski <l.skalski@samsung.com>
Date: Tue, 22 Apr 2014 11:11:20 +0200
Subject: [PATCH] 0.113: polkitd: Fix problem with removing non-existent source

Bug: https://bugs.freedesktop.org/show_bug.cgi?id=77167
Applied-upstream: 0.113, commit:3ca4e00c7e003ea80aa96b499bc7cd83246d7108

Imported from policykit-1_0.105-31.1.debian.tar.xz

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
 src/polkitd/main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/polkitd/main.c b/src/polkitd/main.c
index b21723f697ce..f18fb917c3b6 100644
--- a/src/polkitd/main.c
+++ b/src/polkitd/main.c
@@ -93,7 +93,7 @@ on_sigint (gpointer user_data)
 {
   g_print ("Handling SIGINT\n");
   g_main_loop_quit (loop);
-  return FALSE;
+  return TRUE;
 }
 
 int
